lib/galaxy/model/migrations/alembic/versions_gxy/c63848676caf_update_username_column_schema_and_data.py

```diff
@@ -0,0 +1,53 @@
+"""Update username column schema and data
+
+Revision ID: c63848676caf
+Revises: c14a3c93d66b
+Create Date: 2024-06-11 13:41:36.411803
+
+"""
+
+from alembic import op
+from sqlalchemy import (
+    or_,
+    select,
+    update,
+)
+
+from galaxy.managers.users import username_from_email_with_connection
```

Comment:
Can't import from managers here, that breaks the package structure. Generally speaking it's probably wise to have a fixed copy in the migration so we don't alter migration results accidentally.